<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>621</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>791</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simulation Loop</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>711</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>400</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>START Simulation</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_2" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>711</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>400</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET Simulation</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>310</y>
      <width>201</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Simulation</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Time [ms]:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="timeSpinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Period [ms]:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="periodSpinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>959</x>
      <y>9</y>
      <width>201</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Identification parameters</string>
    </property>
    <widget class="QTextEdit" name="thetaTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Theta</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="nomDegSpinBox">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>140</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="forgettingFactorSpinBox">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>260</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000100000000000</double>
     </property>
     <property name="value">
      <double>0.990000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="denomDegSpinBox">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>170</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="delayBoxSpinBox">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>200</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="histLenSpinBox">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>230</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>6000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nominator degree</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Denominator degree</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Delay</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Treshold</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Forgetting factor</string>
     </property>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="simulationTree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>221</width>
      <height>251</height>
     </rect>
    </property>
    <property name="indentation">
     <number>15</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="headerCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>110</number>
    </attribute>
    <column>
     <property name="text">
      <string>Node Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Object Type</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="propertyTree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>221</width>
      <height>321</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <property name="indentation">
     <number>15</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="headerDefaultSectionSize">
     <number>110</number>
    </attribute>
    <column>
     <property name="text">
      <string>Property Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>customPlot</zorder>
   <zorder>startButton</zorder>
   <zorder>customPlot_2</zorder>
   <zorder>resetButton</zorder>
   <zorder>groupBox</zorder>
   <zorder>simulationTree</zorder>
   <zorder>propertyTree</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1170</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRegulation_Simulator">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionLoad_Parameters"/>
    <addaction name="actionSave_Parameters"/>
   </widget>
   <addaction name="menuRegulation_Simulator"/>
  </widget>
  <action name="actionLoad_Parameters">
   <property name="text">
    <string>Load Parameters</string>
   </property>
  </action>
  <action name="actionSave_Parameters">
   <property name="text">
    <string>Save Parameters</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
